configs/cli/config.toml

```diff
@@ -0,0 +1,26 @@
+[channel]
+  state = ""
+  status = ""
+  topic = ""
+
+[filter]
+  contact = ""
+  email = ""
+  limit = ""
+  metadata = ""
+  name = ""
+  offset = ""
+  raw_output = ""
+
+[remotes]
+  bootstrap_url = "http://localhost"
+  certs_url = "http://localhost"
+  http_adapter_url = "http://localhost/http"
+  msg_content_type = ""
+  reader_url = "http://localhost"
+  things_url = "http://localhost"
+  tls_verification = false
+  users_url = "http://localhost"
+
+[user_token]
+  user_token = ""
```

Comment:
@WashingtonKK

`build/mainflux-cli config  user_token "123"`  This command is not working